internal/ui/ui_glfw.go

```diff
@@ -200,38 +196,102 @@ func initialize() error {
 	return nil
 }
 
-type monitor struct {
+func setInitMonitor(m *glfw.Monitor) {
+	theUI.initMonitor = m
+	theUI.initDeviceScaleFactor = theUI.deviceScaleFactor(m)
+	// GetVideoMode must be called from the main thread, then call this here and record
+	// initFullscreen{Width,Height}InDIP.
+	v := m.GetVideoMode()
+	theUI.initFullscreenWidthInDIP = int(theUI.dipFromGLFWMonitorPixel(float64(v.Width), m))
+	theUI.initFullscreenHeightInDIP = int(theUI.dipFromGLFWMonitorPixel(float64(v.Height), m))
+}
+
+// Monitor is a wrapper around glfw.Monitor.
+type Monitor struct {
 	m  *glfw.Monitor
 	vm *glfw.VidMode
 	// Pos of monitor in virtual coords
-	x int
-	y int
+	x      int
+	y      int
+	width  int
+	height int
+	id     int
+	name   string
+}
+
+// Bounds returns the monitor's bounds.
+func (m *Monitor) Bounds() image.Rectangle {
+	return image.Rect(m.x, m.y, m.x+m.width, m.y+m.height)
+}
+
+// Name returns the monitor's name.
+func (m *Monitor) Name() string {
+	return m.name
+}
+
+// ID returns the monitor's id.
+func (m *Monitor) ID() int {
```

Comment:
I don't think ID method is needed.

Reply:
With the refactors listed above, this is correct.